latest/index.bs

```diff
@@ -190,283 +190,291 @@ OME-NGFF tables are compatible with the [AnnData model](https://github.com/scver
     |
     ├── .zattrs               # `.zattrs` of any group containing a table MUST contain `"tables"`, which is an array with the keys of the subgroubs that are a table.
```

Comment:
can this be removed?

Reply:
Yes, I think line 191 is wrong now. That will be `.zattrs` of the multiscales Image.